src/main/webapp/game.js

```diff
@@ -1,27 +1,35 @@
 const APP_ID = "1024158";
 const CLIENT_KEY = "d15fbbe1c77552dc5097";
 const PUSHER_APPLICATION_NAME = "song-guessing-game";
-const PUSHER_CHAT_CHANNEL_NAME = "chat-update";
+const PUSHER_CHAT_CHANNEL_NAME_BASE = "chat-update-";
 const CSS_MESSAGE_CLASS_DICT = {
   guess: "",
   spectator: "message-spectator",
   correct: "message-correct",
   announcement: "message-announcement",
 };
+const ROOM_ID = getRoomId();
 // TODO: @salilnadkarni, replace with userid from cookie (in datastore)
 const USER_ID = "_" + Math.random().toString(36).substr(2, 9);
 var videoId = "";
 
+function getRoomId() {
```

Comment:
What do you think about adding this to common-functions.js as well? Seems useful to use across scripts?

Reply:
Yup, sounds good